rollup.build.js

```diff
@@ -20,13 +21,19 @@ const BANNER = `/**!
 * MIT License
 */`
 
+const NAMESPACE_PREFIX = process.env.NAMESPACE || 'tippy'
+
 const extensions = ['.js', '.ts']
 
 const plugins = {
   babel: babel({
     exclude: 'node_modules/**',
     extensions,
   }),
+  replace: replace({
+    "NAMESPACE_PREFIX = 'tippy'": `NAMESPACE_PREFIX = '${NAMESPACE_PREFIX}'`,
```

Comment:
This could be easier to read with something like `"__NAMESPACE_PREFIX__": NAMESPACE_PREFIX,`

Then in the code it could be referenced like this:

```javascript
export const IOS_CLASS = '__NAMESPACE_PREFIX__-iOS'
```
The big benefits here is
*  it looks very obvious when viewing the source that `__NAMESPACE_PREFIX__ is a value that gets replaced at build time. As opposed to `NAMESPACE_PREFIX` which just looks like a standard javascript variable.
* there's less variables in the built code that ships to the browser. If someone were to look at the built version of tippy, they'd see `export const IOS_CLASS = 'tippy-iOS'`

Reply:
Yeah I avoided it because of tests but I've updated them now 🙂